ffi/derive/src/convert.rs

```diff
@@ -134,7 +134,10 @@ fn derive_ffi_type_for_fieldless_enum(
     repr: &[syn::NestedMeta],
 ) -> TokenStream {
     if enum_.variants.len() == 1 {
-        abort!(enum_name, "one-variant enums have representation of ()");
+        abort!(
+            enum_name,
+            "one-variant enums have representation of (), try `#[ffi_type(opaque)]`"
```

Comment:
```suggestion
            "one-variant fieldless enums have representation of (), try `#[ffi_type(opaque)]`"
```

as opposed to data-carrying enums